File: Api/NotatnikMechanika.Shared/Models/User/CreateUserModel.cs
﻿using System.ComponentModel.DataAnnotations;

namespace NotatnikMechanika.Shared.Models.User
{
    public class CreateUserModel
    {
        [Required]
        public string Password { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        [EmailAddress]
        [Required]
        public string Email { get; set; }
    }
}

File: Clients/NotatnikMechanika.Flutter/lib/features/domain/entities/user/create_user_model.dart
import 'package:meta/meta.dart';

class CreateUser {
  String password;
  String name;
  String surname;
  String email;

  CreateUser({
    @required this.password,
    @required this.name,
    @required this.surname,
    @required this.email,
  });
}
